### ISSUE TITLE
Make sure format bar on post screen is always sticky above the keyboard

### ISSUE THREAD

**Issue by alisonuxdesign** [ezl/dailyidea.com#507]
MOBILE ONLY: Please make sure that in portrait view on mobile that the format bar is always sticky above the keyboard also please make the "enter" key be the post key. We will have no post button. 

![post_portrait](https://user-images.githubusercontent.com/63747360/89449768-c73ae700-d716-11ea-91a0-a584ddd4e930.png)

Please note that if possible on landscape we don't want the format bar to be sticky. It gets in the way too much. 

![post_landscape](https://user-images.githubusercontent.com/63747360/89449778-cbff9b00-d716-11ea-80ea-db38a01fc6b0.png)

**Comment by vedmant** [comment]
@alisonuxdesign How this will work with current fixed bottom bar? For example this is how it shows currently in the emulator:
<img width="421" alt="Screenshot at Sep 28 23-18-18" src="https://user-images.githubusercontent.com/5052406/94481756-e7e86000-01e0-11eb-9138-c0bca33ac887.png">

Should I hide bottom navigation bar on this page?

Also about changing text of "enter" key, it doesn't look like it's possible, I couldn't find anything about changing it from the browser. This might be possible only in native mobile app. Also if I change return key to post key, there is not be a way to add multi line text in editor. So we need submit button somewhere.

**Comment by danleyb2** [comment]
@alisonuxdesign Have been able to stick the format icons on top of the bottom toolbar but it's not possible to use the KeyBoard buttons as @vedmant  has noted.

This is how it looks with the submit button hidden too

https://user-images.githubusercontent.com/9919961/94759383-68a58880-03a8-11eb-83d9-02e984aab580.jpg

**Comment by alisonuxdesign** [comment]
To anwer a few questions: 

@vedmant the bottom fixed nav will be hidden just on this page. I also found a few articles for how to change the mobile keyboard return key: https://stackoverflow.com/questions/1417702/change-return-key-in-an-iphone-webapp-textbox. I had a convo with @f-prime a while back and we did determine that this was possible. 

@danleyb2 and @vedmant if this is a huge technical hurdle with the format bar my vote is that we just get rid of it. it doesn't need to be part of this version and right now its more of an obstacle to the user experience than an addition
